apps/web/src/components/admin/AdminRegistrantsTable.tsx

```diff
@@ -318,9 +318,7 @@ export const AdminRegistrantsTable = () => {
                   <td className='px-6'>{reg.email}</td>
                   <td className='px-6'>{mapEnumData(reg.specialty, Specialty)}</td>
                   <td className='px-6'>
-                    {!reg.deploymentLocations?.length
-                      ? 'Any'
-                      : mapEnumData(reg.deploymentLocations, Location)}
+                    {!reg.deploymentLocations?.length ? 'Any' : reg.deploymentLocations.join(',')}
```

Comment:
Is this intended?

Vancouver,Richmond,AgassizHarrison,TriCities,Langley,SouthSurreyWhiteRock,CowichanValleySouth,Oceanside,GreaterNanaimo,GreaterVictoria,ComoxValley,Fernie,Kimberley,Creston,SouthernOkanagan,Penticton,Revelstoke,Kamloops,UpperSkeena,PrinceRupert,Stikine

Reply:
I can make this more elegant, but yes it was intended. FNHA locations were not being displayed, and joining the string array fixed the issue.